lib/krane/version.rb

```diff
@@ -1,4 +1,4 @@
 # frozen_string_literal: true
 module Krane
-  VERSION = "2.1.2"
+  VERSION = "2.1.2.a"
```

Comment:
In general, we bump the version in a dedicated PR (easier history, gives us a moment to ensure the CHANGELOG, etc is up to date), and I would recommend simply bumping the patch version to 3 instead of `.a`.